src/ocean/application/components/GCStats.d

```diff
@@ -39,6 +39,9 @@ public class GCStats
     }
     else
     {
+        pragma(msg, "There will be no gc stats! You need to use dmd transitional:");
+        pragma(msg, "https://github.com/sociomantic-tsunami/dmd-transitional");
+
```

Comment:
This whole module requires `GC.monitor` to make sense right? So this will only trigger a message for people really depending on `GC.monitor`?

Reply:
yes